jekyll/_cci2/remote-docker.md

```diff
@@ -64,30 +64,28 @@ Let’s break down what’s happening during this build’s execution:
 - All docker-related commands are executed in your main container, too, but building/pushing images and running containers happens in the remote Docker Engine.
 - We use project environment variables to store credentials for Docker Hub.
 
-## Separation of environments
+## Separation of Environments
+Since the [job space][job-space] and [remote docker]({{ site.baseurl }}/2.0/glossary#remote-docker) are separated environments, there's one caveat: containers running in your job space can’t directly communicate with containers running in remote docker.
 
-Since [job space]({{ site.baseurl }}/2.0/glossary#job-space) and [remote docker]({{ site.baseurl }}/2.0/glossary#remote-docker) are two separated environments there's one caveat. Containers, running in your job space, cannot directly communicate with containers, running in remote docker. A few practical examples and ways work around:
+### Examples and Workarounds
+It’s impossible to start a service in remote docker and ping it directly from a main container (and vice versa). To solve that, you’ll need to interact with a a service from remote docker, as well as through the same container:
```

Comment:
Each example/workaround should be a bullet or have a title for separation, otherwise they are smashed together